vm/systemSmartContracts/esdt.go

```diff
@@ -877,6 +1062,11 @@ func (e *esdt) unSetSpecialRole(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return returnCode
 	}
 
+	if isCreateNFTRoleInArgs(args.Arguments[2:]) {
```

Comment:
So, create NFT role could not be unSet ?

Reply:
exactly. It can be transferred to another owner or stopped forever

Reply:
ok